tests/integration/BuildsHttpRequests.php

```diff
@@ -34,21 +33,10 @@ protected function requestWithJsonBody(Request $req, array $json): Request
 
     protected function requestAsUser(Request $req, int $userId): Request
     {
-        $token = Str::random(40);
+        $token = AccessToken::generate($userId);
```

Comment:
What's the reasoning behind this change? Franz purposefully wanted to avoid this in https://github.com/flarum/core/commit/31765388c1eec0aba090514633111c2a946ac257 (although why I'm not sure)